### ISSUE TITLE
Missing functionality in Keras TensorBoard callback 

### ISSUE THREAD

**Issue by Hvass-Labs** [tensorflow/tensorflow#16774]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No.
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: Binary.
- **TensorFlow version (use command below)**: 1.5.0 (Keras 2.1.2-tf)
- **Python version**: 3.6
- **Bazel version (if compiling from source)**: NA
- **GCC/Compiler version (if compiling from source)**: NA
- **CUDA/cuDNN version**: 9.0 / 7.0
- **GPU model and memory**: GTX 1070 8GB
- **Exact command to reproduce**: NA

### Describe the problem

The TensorBoard-callback in Keras supports the writing of embeddings to the log, but this is not supported in the TensorFlow version of Keras, even though the doc-string of the TensorFlow version actually lists these parameters.

https://github.com/tensorflow/tensorflow/blob/579125e87af201ae6b6fa872b6dc3f3ecb400de9/tensorflow/python/keras/_impl/keras/callbacks.py#L643-L651

But these are missing from the `__init__`:

https://github.com/tensorflow/tensorflow/blob/579125e87af201ae6b6fa872b6dc3f3ecb400de9/tensorflow/python/keras/_impl/keras/callbacks.py#L656-L662

This is the original Keras implementation which has e.g. `embeddings_freq`:

https://github.com/keras-team/keras/blob/ad00676b80556a6354180a1bfa3009d4db316d3e/keras/callbacks.py#L642-L650

There seems to be a difference between the original Keras implementation and the version in TensorFlow. I am wondering if these are somehow developed in parallel so it is actually not the original Keras that is included with TensorFlow?

Thanks!

**Comment by fchollet** [comment]
This functionality currently relies on feature from `contrib`, while `tf.keras` is in core TF. Thus it cannot be support by `tf.keras` yet.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 108 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 123 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 138 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 153 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 168 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 183 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 198 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 213 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by harshini-gadige** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
